src/test/java/core/basesyntax/service/impl/CsvFileWriterServiceImplTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.service.CsvFileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvFileWriterServiceImplTest {
+    private static CsvFileWriterService csvFileWriterService;
+    private static final String VALID_FILE_PATH = "src//test//resources//Report.csv";
+    private static final String INVALID_FILE_PATH = "nonexistent-directory/nonexistent-file.txt";
+
+    @BeforeAll
+    static void setUp() {
+        csvFileWriterService = new FileWriterServiceImpl();

```

Comment:
The class is named FileWriterServiceImp and test is named `Csv`FileWriterServiceImplTest . The naming of test class and original class should be consistent.